pureconfig-argonaut/src/main/scala/pureconfig/contrib/implicits/package.scala

```diff
@@ -0,0 +1,22 @@
+package pureconfig.contrib.implicits
+
+import scalaz._
+import scalaz.Validation.FlatMap._
+import argonaut._, Argonaut._
+import com.typesafe.config.{ConfigFactory, ConfigRenderOptions, ConfigValue}
+import pureconfig.ConfigConvert
+
+import scala.reflect.ClassTag
+import scala.util.Try
+
+object `package` {
+  implicit def configConvertFromArgonaut[T: EncodeJson : DecodeJson](implicit ct: ClassTag[T]): ConfigConvert[T] = new ConfigConvert[T] {
+    def from(config: ConfigValue): Try[T] = Try(
+      config.render(ConfigRenderOptions.concise())
```

Comment:
Maybe you want to `setJson(true)` in the `ConfigRenderOptions`? I don't think argonaut can parse HOCON and the default should be HOCON for [`render`](https://typesafehub.github.io/config/latest/api/com/typesafe/config/ConfigValue.html#render--).